packages/shortest/src/browser/integrations/index.ts

```diff
@@ -1,3 +1,2 @@
-import { GitHubTool } from "./github";
-
-export { GitHubTool };
+export { GitHubTool } from './github';
+export { MailosaurTool } from './mailosaur';
```

Comment:
index file to export all the integrations that will be used in `browser-tools.ts`